CelesteNet.Client/Components/CelesteNetPlayerListComponent.cs

```diff
@@ -601,34 +644,67 @@ public CelesteNetPlayerListComponent(CelesteNetClientContext context, Game game)
         protected override void Render(GameTime gameTime, bool toBuffer) {
             PrepareRenderLayout(out float scale, out float y, out Vector2 sizeAll);
 
-            float x = 25f * scale;
-            float sizeAllXPadded = sizeAll.X + 50f * scale;
-            float chatStartY = (Context?.Chat?.RenderPositionY ?? UI_HEIGHT) - 5f;
+            float x = Margin * scale;
+            float sizeAllXPadded = sizeAll.X + 2 * PaddingX * scale;
+            float chatStartY = (Context?.Chat?.RenderPositionY ?? UI_HEIGHT) - ChatOffset;
             Color colorFull = Color.Black * 0.8f;
             Color colorFaded = Color.Black * 0.5f;
 
             switch (ListMode) {
                 case ListModes.Classic:
-                    SplitRectAbsolute(x, y - 25f * scale, sizeAllXPadded, sizeAll.Y + 30f * scale, chatStartY, colorFull, colorFaded);
+                    SplitRectAbsolute(
+                        x, y,
+                        sizeAllXPadded, sizeAll.Y + 2 * PaddingY * scale,
+                        chatStartY,
+                        colorFull, colorFaded
+                    );
                     break;
 
                 case ListModes.Channels:
                     // own channel box always there
-                    SplitRectAbsolute(x, y - 25f * scale, sizeAllXPadded, SizeUpper.Y + 30f * scale, chatStartY, colorFull, colorFaded);
+                    SplitRectAbsolute(
+                        x, y,
+                        sizeAllXPadded, SizeUpper.Y + 2 * PaddingY * scale,
+                        chatStartY,
+                        colorFull, colorFaded
+                    );
+
+                    // skip below the drawn rect and include a gap
+                    float columnY = y + SizeUpper.Y + 2 * PaddingY * scale + SplitGap * scale;
 
                     if (SplitViewPartially && SplitSuccessfully) {
                         // two rects for the two columns
-                        float sizeColXPadded = SizeColumn.X + 25f * scale;
-                        SplitRectAbsolute(x, y + SizeUpper.Y + 15f * scale, sizeColXPadded - 5f * scale, sizeAll.Y - SizeUpper.Y + 30f * scale, chatStartY, colorFull, colorFaded);
-                        x += sizeColXPadded + 5f * scale;
-                        SplitRectAbsolute(x, y + SizeUpper.Y + 15f * scale, sizeAllXPadded - sizeColXPadded - 5f * scale, sizeAll.Y - SizeUpper.Y + 30f * scale, chatStartY, colorFull, colorFaded);
+                        float sizeColXPadded = SizeColumn.X + PaddingX * scale;
```

Comment:
Technically this needs to be with `-SplitGap/2` ~~and then in line 690 the use of SplitGap needs to also be `/2`~~ otherwise the Gap is left-aligned at the center-line instead of centered. I think the thing I struck out is wrong but would be good to double check :thinking: